### PR TITLE
CT-2439 Add translations for clearing response

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Add translations and remove strings within Clear responses form
 
## Self-review checklist

* [x] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Screenshots
image

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2439
 
### Deployment
n/a
 
### Manual testing instructions
Create a case with a response pending Press Office Clearance, then log in as Press Office and clear the response, view the form to ensure it all looks Ok.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -526,6 +526,7 @@ en:
       case-details: Case details
       original-case-details: Original case details
       clear_response: Clear response
+      clearance_copy: "You are clearing the response to: "
```
Comment (mseedat-moj) [inline config/locales/en.yml]: No need for double quotes?

Comment (digitalali-moj) [reply config/locales/en.yml]: Changed to single quote

**Review by mseedat-moj — APPROVED** [review]
Good work

**Comment by digitalali-moj** [comment]
Will merge into CT-2154

**Comment by digitalali-moj** [comment]
Moved to https://github.com/ministryofjustice/correspondence_tool_staff/pull/980 to rebase against master instead

**Pull request closed without merging**